ngraph/test/backend/adaptive_avg_pool.in.cpp

```diff
@@ -0,0 +1,131 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "util/engine/test_engines.hpp"
+#include "util/test_case.hpp"
+#include "util/test_control.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+static string s_manifest = "${MANIFEST}";
+using TestEngine = test::ENGINE_CLASS_NAME(${BACKEND_NAME});
+
+NGRAPH_TEST(${BACKEND_NAME}, adaptive_avg_pool_1d)
+{
+    auto data = make_shared<op::Parameter>(element::f32, Shape{2, 3, 7});
```

Comment:
I think test with i32 and i64 data should be added. Same for adaptive max pool.

Reply:
Not torch nor paddlepaddle have implementations for int32 and int64 types. Not sure if we need to test it.

Reply:
Ok, I must have checked wrong documentation.